src/views/dataset/MultiSourceConfiguration.vue

```diff
@@ -937,6 +984,24 @@ export default class MultiSourceConfiguration extends Vue {
       return null;
     }
   }
+
+  get submitError(): string | null {
+    if (!this.submitEnabled()) {
+      return "Not all variables are assigned";
+    }
+
+    if (!this.isRGBAssignmentValid) {
+      return "If splitting RGB file into channels, then filenames must be assigned to another variable";
+    }
+    return null;
```

Comment:
Maybe return a `Unknown error` ?

Reply:
Actually, if it gets this far, it means there is no error to display.